### PR TITLE
Fix about variable order in Italian translation

### PR DESCRIPTION
**Pull request by bissim**

### CODE REVIEW

**Comment by Crizz0** [comment]
@Mauron Would you fix the conflict with a rebase, pls?

**Comment by bissim** [comment]
Hi @Crizz0,
At the moment I've got my hands full to perform a rebase.
I'll try tomorrow.

**Comment by Crizz0** [comment]
It should be okay if you just do:

> git checkout it-translation
> git fetch orgin
> git merge orgin/develop-ascraeus
> git push Mauron it-translation

Try it out :) Maybe you have to fix a conflict before pushing, but git will give you a notice. 

Alternative: http://www.git-scm.com/book/en/v2/Git-Branching-Rebasing

**Comment by bissim** [comment]
@Crizz0 I tried but I think this made me rebase my own fork with no actual fetch from the original repository.

**Comment by bissim** [comment]
Yes!

File: language/it/movemessage.php [hunk 1]
```diff
@@ -17,5 +17,6 @@
 }
 
 $lang = array_merge($lang, array(
-	'MOVED_MESSAGE'				=> 'Spostato dal forum <strong>%1$s</strong> al forum <strong>%2$s</strong> da <strong>%4$s</strong> il %3$s',
+<<<<<<< HEAD
```
Comment (Crizz0) [inline language/it/movemessage.php]: That is not supposed to be here! Please check your code and remove those leftovers.

**Comment by bissim** [comment]
FU-

**Pull request merged**